src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -656,6 +656,11 @@ private string ParseGeoMasterName(string geomasterHostName)
         {
             string geoMasterName = null;
 
+            if (!string.IsNullOrWhiteSpace(geomasterHostName) && !geomasterHostName.StartsWith(Uri.UriSchemeHttp, StringComparison.CurrentCultureIgnoreCase))
```

Comment:
Shouldn't we use UriSchemeHttps here? Below we are prepending Https but here we are checking using Http.

Reply:
For our particular purposes it doesn't matter if its HTTPs or HTTP.